### ISSUE TITLE
Missing tree files

### ISSUE THREAD

**Issue by dpny518** [zhaoyi2/cvte_model_funtune#3]
File "steps/libs/nnet3/train/chain_objf/acoustic_model.py", line 378, in check_for_required_files
    raise Exception('Expected {0} to exist.'.format(file))
Exception: Expected exp/chain/tri6b_tree/ali.1.gz to exist.

https://gitee.com/mirrors/kaldi/blob/master/egs/wsj/s5/steps/libs/nnet3/train/chain_objf/acoustic_model.py#L369

**Comment by zhaoyi2** [comment]
hi, you shell use steps/align_fmllr.sh to generate ali files, and copy  ali files to tri6b_tree dir, or you can point the generated address of ali-files  to tri6b_tree dir.
hope to solve your problem

**Comment by dpny518** [comment]
it is ok to use the new data to generate alignment files?

**Comment by zhaoyi2** [comment]
> it is ok to use the new data to generate alignment files?

hi, new data is aligned through the GMM to provide a good starting point for the chain model. you can use the cvte GMM to generated alignment files without training your own GMM, of course you can train your GMM use your new data but need more time.

**Comment by dpny518** [comment]
it seems the alignment isn't really used in the train.py, it is just a check, would it be easier to add a touch $src_tree_dir/ali.1.gz just so it passes checks..we have the final tree already made

**Comment by zhaoyi2** [comment]
> it seems the alignment isn't really used in the train.py, it is just a check, would it be easier to add a touch $src_tree_dir/ali.1.gz just so it passes checks..we have the final tree already made

i agree with you. i just checked the align_fmllr_lats.sh file, which contains the alignment operation.
